reducers/context.js

```diff
@@ -0,0 +1,29 @@
+const {GET_CONTEXT, INCREMENT_CONTEXT_RESETS, ADD_FILE_TO_CONTEXT} = require('../actions/context')
+const initialState = { resets: -1, commands: [], fullCommands: [], files: [] }
+
+function getContext (state = initialState) {
+  return state
```

Comment:
Again, not clear why `getContext` exists here, seems like a leftover from previous refactoring.

Reply:
I'll split context related changes into a separate branch.

Reply:
Done (see `aorinevo/I-1150_master_AN_redux_context`)